src/elements/title/title.visual.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { html } from 'lit';
+
+import { describeViewports, visualDiffDefault } from '../core/testing/private.js';
+
+import '../title.js';
+
+describe(`sbb-title`, () => {
+  describeViewports({ viewports: ['zero', 'small', 'large', 'ultra'] }, () => {
+    for (const level of ['1', '2', '3', '4', '5', '6']) {
+      it(
+        `visual level ${level}`,
+        visualDiffDefault.with(async (setup) => {
+          await setup.withFixture(
+            html`<sbb-title level=${level}>
+              Data without insights are trivial, and insights without action are pointless
```

Comment:
I'd suggest (even when it was not like that in Chromatic) to add a <p> tag below the title with a short random text to cover also the margins which are part of the sbb-title.

Reply:
Good idea